benchmark/util.mjs

```diff
@@ -15,10 +17,10 @@ import baselinePresetEnv from "@babel-baseline/preset-env";
 import baselinePresetTypescript from "@babel-baseline/preset-typescript";
 import baselinePluginCommonjs from "@babel-baseline/plugin-transform-modules-commonjs";
 
-// eslint-disable-next-line import/no-extraneous-dependencies
 import { USE_ESM } from "$repo-utils";
 import { Bench } from "tinybench";
-import { globSync, readFileSync } from "fs";
```

Comment:
According to https://bun.sh/docs/runtime/nodejs-apis#node-fs this should be supported, could you open a bug report?

Reply:
https://github.com/oven-sh/bun/issues/13623
It seems they already have. :)

By the way, can you approve the installation of CodSpeed ​​app? Thanks!
![image](https://github.com/user-attachments/assets/af697105-36c1-4c0a-a32e-482d2288e984)

Reply:
Done :)